common/src/main/java/com/wynntils/models/statuseffects/StatusEffectModel.java

```diff
@@ -87,7 +90,18 @@ public void onTabListCustomization(PlayerInfoFooterChangedEvent event) {
             StyledText prefix = parts.get(0);
             StyledText name = parts.get(1);
             StyledText displayedTime = parts.get(2);
-            newStatusEffects.add(new StatusEffect(name, displayedTime, prefix));
+            StyledText modifier;
+
+            // Split the modifier and name, which are separated by a space
+            StyledText[] modifierStr = name.split("\\s{1}");
```

Comment:
```suggestion
            StyledText[] modifierStr = name.split("\\s");
```

Isn't this the same?

Reply:
Oh right, I was just blindly copying what was above